rules/windows/defense_evasion_masquerading_suspicious_werfault_childproc.toml

```
@@ -0,0 +1,47 @@
+[metadata]
+creation_date = "2020/08/24"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/24"
+
+[rule]
+author = ["Elastic"]
+description = """
+A suspicious WerFault child process was detected. This behavior may indicate an attempt to run unnoticed. Verify process
+details such as commandline, network connections, file writes and parent process details as well.
```

```suggestion
A suspicious WerFault child process was detected, which may indicate an attempt to run unnoticed. Verify process
details such as command line, network connections, file writes and parent process details as well.
```

Will need to re-lint